src/runtime/composables/useFormGroup.ts

```diff
@@ -1,10 +1,10 @@
 import { inject, ref } from 'vue'
 import { type UseEventBusReturn, useDebounceFn } from '@vueuse/core'
-import type { FormEvent, FormEventType } from '../types/form'
+import type { FormEvent, FormEventType, InjectedFormGroupValue } from '../types/form'
 
 export const useFormGroup = () => {
     const formBus = inject<UseEventBusReturn<FormEvent, string> | undefined>('form-events', undefined)
-    const formGroup = inject('form-group', undefined)
+    const formGroup = inject<InjectedFormGroupValue>('form-group')
 
     const blurred = ref(false)
 
```

Comment:
It's a good suggestion @romhml . If we want to avoid overriding the `id` attribute, I think we should also avoid overriding the `name` and `size` attributes. In the current source code, the `name` and `size` attributes can be overridden by `UFormGroup`. It would be good to have the same behavior for attribute assignment.

Reply:
I'd suggest to handle the `id` properly in this PR and open a new one to implement a consistent behaviour for other props to keep things manageable, what do you think?

Reply:
Yes, that's great, @romhml. I will handle it.

Reply:
@romhml updated